operator/src/test/java/org/keycloak/operator/ClusterOperatorTest.java

```diff
@@ -122,6 +137,41 @@ private static void calculateNamespace() {
     namespace = "keycloak-test-" + UUID.randomUUID();
   }
 
+  protected static void deployDB() {
+    // DB
+    Log.info("Creating new PostgreSQL deployment");
+    k8sclient.load(KeycloakDeploymentE2EIT.class.getResourceAsStream("/example-postgres.yaml")).inNamespace(namespace).createOrReplace();
+
+    // Check DB has deployed and ready
+    Log.info("Checking Postgres is running");
+    Awaitility.await()
+            .untilAsserted(() -> assertThat(k8sclient.apps().statefulSets().inNamespace(namespace).withName("postgresql-db").get().getStatus().getReadyReplicas()).isEqualTo(1));
+  }
+
+  // TODO improve this (preferably move to JOSDK)
+  protected void savePodLogs() {
```

Comment:
this forces us to enclose all the tests in big `try ... catch` blocks, it will also increase the technical debt(already pretty high) of the tests.
Since this is not related to the scope of this PR, can you please exclude from the current PR and open a relevant ticket?

Reply:
This is related to this PR as we already had try-catch in the original tests and we were printing it to the log. This change improves this as it at least allows the logs to be saved to a file.

Created:
* #10128
* https://github.com/java-operator-sdk/java-operator-sdk/issues/925

Reply:
👍